README.md

```diff
@@ -1,5 +1,7 @@
 # JLM: A research compiler based on the RVSDG IR
 [![Tests](https://github.com/phate/jlm/actions/workflows/tests.yml/badge.svg)](https://github.com/phate/jlm/actions/workflows/tests.yml)
+[![HLS](https://github.com/phate/jlm/actions/workflows/hls.yml/badge.svg)](https://github.com/phate/jlm/actions/workflows/hls.yml)
+[![Doxygen](https://github.com/phate/jlm/actions/workflows/Doxygen.yml/badge.svg)](https://github.com/phate/jlm/actions/workflows/Doxygen.yml)
```

Comment:
This is new compared to the last review. Could we not try to do everything all at once, please?

Reply:
Removed